azure-functions/acbs-function/acbs-issue-facility/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = df.orchestrator(function* updateACBSfacility(context) {
     const { facilityId, facility, deal } = context.df.getInput();
 
     if (facilityId.includes(CONSTANTS.DEAL.UKEF_ID.PENDING) || facilityId.includes(CONSTANTS.DEAL.UKEF_ID.TEST)) {
-      throw new Error('Invalid facility ID %s', facilityId);
+      throw new Error('Invalid facility ID %O', facilityId);
```

Comment:
Looks like %O works for strings and numbers too, but it would look cleaner with %s